package.json

```diff
@@ -13,25 +13,25 @@
   },
   "scripts": {
     "postinstall": "husky install && js-tools lint:yarn-deduplicate --fix",
-    "build": "turbo run version",
-    "build-size": "yarn build && size-limit",
+    "build": "cross-env NODE_OPTIONS=--openssl-legacy-provider turbo run version",
+    "build-size": "cross-env NODE_OPTIONS=--openssl-legacy-provider yarn build && size-limit",
     "check:types": "yarn tsc && yarn tsc --project cypress",
     "clean": "git clean -fxd -e '.idea' -e '.vscode'",
     "predocs": "rimraf docs",
     "docs": "run-s storybook:build playroom:build",
     "lint": "js-tools lint",
     "lint-changed": "git add . && yarn lint-staged",
-    "preplayroom:build": "yarn playroom:generate",
-    "playroom:build": "NODE_OPTIONS=--max_old_space_size=4096 playroom build",
+    "preplayroom:build": "cross-env NODE_OPTIONS=--openssl-legacy-provider yarn playroom:generate",
+    "playroom:build": "cross-env NODE_OPTIONS=\"--openssl-legacy-provider --max_old_space_size=4096\" playroom build",
```

Comment:
Why do we need `\` in "playroom:build" but not in "preplayroom:build"?

Reply:
let me try without it

Reply:
updated script